docs/remotes.md

```diff
@@ -18,7 +18,7 @@ When running this command, you will be prompted first for the remote you want to
 
 ## `git: push`
 
-This command performs a simple `git push`.  In recent Git versions, this will result in a push to the tracking branch if it exists, else a remote branch with the same name as the local.
+This command will push current branch to the tracking branch. If the tracking branch is not set, you will be asked to configure it.
```

Comment:
The docs is updated to reflect the recent behavior. The original docs is oudated and the command requires a remote tracking branch even without this PR.